### PR TITLE
Week-03 Js home-work

### PR DESCRIPTION
**Pull request by asmafassy**
(JavaScript 1) Week 03 Js Home Work

### CODE REVIEW

**Review by amritpandey — COMMENTED** [review]
well done Asma, as usual  impressed with your code and specially that you used  high order function.

File: javascript/javascript1/week3/item.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+ let deletingNameIndex = names.indexOf(nameToRemove);
```
Comment (aina21) [inline javascript/javascript1/week3/item.js:15]: const

File: javascript/javascript1/week3/notes.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+//Q4) Step 3: Smart-ease - Back to the basics!
+//NOnoN0nOYes (Note taking app)
+
+//-->Save a note
+const notes = [];
+function saveNote(content, id) {
+  if (typeof content === "string" && typeof id === "number") {
+    let note = {
```
Comment (aina21) [inline javascript/javascript1/week3/notes.js:8]: const

File: javascript/javascript1/week3/notes.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+getNote(1); // show id
+console.log(getNote(1));
+
+//-->Log out notes
+let id = 2;
+function logOutNotesFormatted() {
+  for (let i = 0; i < notes.length; i++) {
+    let note = notes[i];
```
Comment (aina21) [inline javascript/javascript1/week3/notes.js:44]: const

File: javascript/javascript1/week3/notes.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    let msg =
```
Comment (aina21) [inline javascript/javascript1/week3/notes.js:45]: const

File: javascript/javascript1/week3/series.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+];
+
+function getPercentage(seriesDuration) {
+  let lifeSpan = 365 * 24 * 80;
```
Comment (aina21) [inline javascript/javascript1/week3/series.js:25]: const

File: javascript/javascript1/week3/series.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+function logOutOfSeriesText() {
+  let totalPercentage = 0.0;
+  for (let i = 0; i < seriesDurations.length; i++) {
+    const series = seriesDurations[i];
+    const daysToHours = series.days * 24;
+    const totalHours = daysToHours + series.hours;
+    const totalHoursInPErcentage = getPercentage(totalHours);
+    let logText = series.title + ' took ' +  totalHoursInPErcentage + '% of my life';
```
Comment (aina21) [inline javascript/javascript1/week3/series.js:36]: const

File: javascript/javascript1/week3/time.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
+//Q2) When will we be there??
+
+  const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function getTravelTime() {
+ let formula = travelInformation.destinationDistance / travelInformation.speed;
```
Comment (aina21) [inline javascript/javascript1/week3/time.js:9]: const

File: javascript/javascript1/week3/time.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+let n = new Date(0, 0);
```
Comment (aina21) [inline javascript/javascript1/week3/time.js:11]: n is not a good name

File: javascript/javascript1/week3/time.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+let timeInSeconds = n.setSeconds(+formula * 60 * 60);
```
Comment (aina21) [inline javascript/javascript1/week3/time.js:12]: const

**Review by aina21 — CHANGES_REQUESTED** [review]
good job :+1:

**Pull request merged**